pom.xml

```diff
@@ -195,6 +197,30 @@
                 <scope>provided</scope>
             </dependency>
 
+            <dependency>
```

Comment:
Just want to voice some unease.  Not necessarily a call for action now.  How sustainable it is to keep pushing every dependency to the top level DM?  Technically it works, but it is poor for information hiding.

Wondering if when we have a filter with some dependencies that are scoped exclusively to it, maybe the responsibility for the DM should be with it.  We'd need to think about the requirements of integration and system tests which may need the same dependency.